val.py

```diff
@@ -264,7 +266,7 @@ def run(
 
             # Perform database operations using the 'session'
             # The session will be automatically closed at the end of this block
-            with db_config.get_session() as session:
+            with db_config.managed_session() as session:
```

Comment:
This I forgot to change. We use managed_session now thats a wrapper over get_session